inc/options/class-wpseo-option.php

```diff
@@ -417,7 +417,12 @@ public function get_option( $options = null ) {
 	 */
 	public function register_setting() {
 		if ( WPSEO_Capability_Utils::current_user_can( 'wpseo_manage_options' ) ) {
-			register_setting( $this->group_name, $this->option_name );
+			if ( $this->multisite_only !== true ) {
```

Comment:
Please consider inverting the first if and getting rid of the `elseif` to improve readability and make use of early returns.